runner.py

```diff
@@ -13,6 +15,9 @@ def get_letter(prompt: str) -> int:
       if len(letter) != 1:
         continue
       letter = letter.lower()
+      # Bypass for hints
+      if ord(letter) == ord('?'):
```

Comment:
How does the player/user know this exists? We should probs add a note in the readme or add something to argparse